Projects/Python/FantasyBaseballTrends.md

```diff
@@ -0,0 +1,11 @@
+# Fantasy Baseball Lineup Helper
```

Comment:
Add project link

Reply:
I addressed the issues in a new pull request with the branch hw2_branch-2.